src/Controller/ArticleFormController.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Article;
+use App\Entity\Image;
+use App\Form\ArticleFormType;
+use App\Form\ImageFormType;
+use App\Repository\ImageRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\File\Exception\FileException;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use App\Service\IdsListGenerator;
+use App\Service\ImageValidatorNAdder;
+
+class ArticleFormController extends AbstractController
+{
+    /**
+     * @Route("/article_form/", name="article_form")
+     */
+    public function index(
+        Request $request,
+        EntityManagerInterface $entityManager,
+        ImageRepository $imageRepository,
+        IdsListGenerator $idsListGenerator,
+        ImageValidatorNAdder $imageValidatorNAdder
+    ): Response {
+        $image = new Image();
+        $imageForm = $this->createForm(ImageFormType::class, $image);
+        $article = new Article();
+        $articleForm = $this->createForm(ArticleFormType::class, $article);
+        $imageIds = '';
```

Comment:
This is useless (imageIds are modified after without being read)

Reply:
Tell that to stan